optimus/devices.go

```diff
@@ -0,0 +1,379 @@
+// The whole module is a piece of shit. Do we really need such kind of
+// benchmark madness?
+
+package optimus
+
+import (
+	"errors"
+	"math"
+
+	"github.com/montanaflynn/stats"
+	"github.com/sonm-io/core/insonmnia/benchmarks"
+	"github.com/sonm-io/core/proto"
+)
+
+const (
+	minCores = 0.01
+	minRAM   = 4 * (1 << 20)
```

Comment:
this constants already defined into `proto/ask_plan.go`

Reply:
It's private.